.markdownlint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "MD013": false,
```

Comment:
This is so impenetrable, are there any more-readable keys for these values? 😭

Reply:
I know, right? I woulda commented them, but .... json. 

Seems like I MIGHT be able to use more verbose rules: https://github.com/DavidAnson/markdownlint/blob/master/.markdownlint.json

I'll play around and see what I can do.